src/sentry/utils/email/manager.py

```diff
@@ -33,7 +33,9 @@ def get_email_addresses(
         for option in (o for o in options if o.value and not is_fake_email(o.value)):
             user_id_emails.append(UserIdEmailArgs(user_id=int(option.user_id), email=option.value))
 
-        user_id_emails_exists = user_service.verify_user_emails(user_id_emails=user_id_emails)
+        user_id_emails_exists = user_service.verify_user_emails(
+            user_id_emails=user_id_emails, only_verified=only_verified
```

Comment:
You can't add a parameter to an RPC method and use it in the same deployment. Instead the new method signature needs to be deployed then callers can be updated.

Deploying a signature change and the callers in the same deploy results in RPC failures for the duration of the deploy process.

Reply:
ahhh i see, yes lemme correct this